invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/formatters.js

```diff
@@ -34,7 +34,10 @@ function formatDocumentToTableView(document, seriesPid = null) {
     serialized['Authors'] = document.metadata.authors;
     if (document.metadata.circulation) {
       serialized['Requests'] = document.metadata.circulation.pending_loans;
-      serialized['Items'] = document.metadata.circulation.has_items_for_loan;
+      serialized['Available Items'] =
+        document.metadata.circulation.has_items_for_loan;
+      serialized['# Loans'] =
```

Comment:
what does `#` mean?

Reply:
Its a common way to indicate its a number